WarehouseManagement.Core/DTOs/Delivery/DeliveryDto.cs

```diff
@@ -8,10 +8,12 @@ public class DeliveryDto
     public string TruckNumber { get; set; } = string.Empty;
     public string Cmr { get; set; } = string.Empty;
     public DateTime DeliveryTime { get; set; }
+    public DateTime? ApprovedOn { get; set; }
     public int Pallets { get; set; }
     public int Packages { get; set; }
     public int Pieces { get; set; }
-    public bool IsApproved { get; set; }
+    public int EntriesWaitingProcessing { get; set; }
+    public int EntriesFinishedProcessing { get; set; }
```

Comment:
is it possible to have something like that instead?
```
entriesWaitingProcessing: {
   "pallets": 20,
   "packages": 50,
   "pieces": 10
},
entriesFinishedProcessing: {
   "pallets": 5,
   "packages": 3,
   "pieces": 1
}
```

I'm sorry that i say this now, but i just aboit it when i saw this.
The reason is so we can give the most information ot the user as possible
![image](https://github.com/user-attachments/assets/a29895ce-bb75-4878-84d2-6281009da67c)

Reply:
I am getting on this

Reply:
it turned out that this would not be so easy,
I recommend assign it as a new task, and if there are no other comments to merge the PR.

Reply:
sure, here is the task https://github.com/orgs/InterCarsBulgaria-TechLeadAcademy/projects/3?pane=issue&itemId=71638102